src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoChildFactoryBase.cs

```diff
@@ -321,5 +322,10 @@ public override string GetNodeCustomName(object smoObject, SmoQueryContext smoCo
         {
             return string.Empty;
         }
+
+        public override string GetNodePathName(object smoObject)
+        {
+            return (smoObject as NamedSmoObject).Name;

```

Comment:
This doesn't work for tables that need a customized name (like temporal tables), because it will omit the schema. Looks like this method needs to be overridden in `TablesChildFactory` and `TableChildFactory`.